app/actions/session.jsx

```diff
@@ -0,0 +1,19 @@
+import Alt from 'alt_flux';
+import { createActions } from 'alt-utils/lib/decorators';
+import sessionSource from 'sources/session';
+
+@createActions(Alt)
+export default class SessionActions {
+  create(user) {
+    return (dispatch) => {
+      sessionSource.create(user).then(result => dispatch(result));
+    };
+  }
+
+  destroy(user) {
+    return (dispatch) => {
+      sessionSource.delete(user);
```

Comment:
just, in order to keep consistency, what would you say if we try to use `destroy` and `destroy` nor `delete` and `delete` ?